x-pack/legacy/plugins/triggers_actions_ui/index.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Legacy } from 'kibana';
+import { Root } from 'joi';
+import { resolve } from 'path';
+
+export function triggersActionsUI(kibana: any) {
+  return new kibana.Plugin({
+    id: 'triggers_actions_ui',
```

Comment:
Should the plugin name be "triggers and actions" or "alerts and actions" like the UI labels?